lib/src/model/study/study.dart

```diff
@@ -9,27 +10,68 @@ import 'package:lichess_mobile/src/model/user/user.dart';
 part 'study.freezed.dart';
 part 'study.g.dart';
 
-@Freezed(fromJson: true)
+@freezed
 class Study with _$Study {
   const Study._();
 
   const factory Study({
     required StudyId id,
     required String name,
-    required ({StudyChapterId chapterId, String path}) position,
     required bool liked,
     required int likes,
     required UserId? ownerId,
-    @JsonKey(fromJson: studyFeaturesFromJson) required StudyFeatures features,
+    required StudyFeatures features,
     required IList<String> topics,
     required IList<StudyChapterMeta> chapters,
     required StudyChapter chapter,
+    // Hints to display in "gamebook"/"interactive" mode
```

Comment:
triple slash :)

Reply:
done, hope I will stop forgetting this at some point :D